### PR TITLE
chore: Remove NDK constraint

### PR DESCRIPTION
**Pull request by BenjaminHalko**
A plugin says it requires NDK version `26.1.10909125`, so the NDK version has been changed from `flutter.ndkVersion` to `26.1.10909125`

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
I don't know if this is right. From what I saw building still worked

**Comment by BenjaminHalko** [comment]
Yep, building still works, but you get this error:
image

**Comment by BenjaminHalko** [comment]
What does prevent building is the fact that `Patch options fields` was missing the `patch_options_viewmodel` import, which prevented building. So that was fix here as well. (Maybe could have made a separate PR or pushed that directly to dev though)

**Comment by BenjaminHalko** [comment]
If this PR gets approved quickly, I'll leave that extra fix in there, otherwise I should probably push the `patch_options_viewmodel` fix to dev so that Manager can build properly.

**Comment by oSumAtrIX** [comment]
> Yep, building still works, but you get this error:

Which plugin causes this. This wasn't an issue before.

**Comment by BenjaminHalko** [comment]
It might have happened after upgrading `intl` to `0.19.0` which is necessary to use `flutter_localizations`, but it is hard to say for certain. It has been happening for at least a month judging from the build logs, but any further and the logs have been deleted.

**Comment by oSumAtrIX** [comment]
A git bisect should be helpful finding the commit, or it may be that plugin, even though it's actually a dependency. I don't know if it's correct practice to simply just pin the ndk version. It is set to flutter.ndkVersion by default for a reason

**Comment by BenjaminHalko** [comment]
The other option is to simply get rid of `ndkVersion` all together and it will instead use the NDK version provided by the `AGP`. This gets rid of the error.

**Comment by oSumAtrIX** [comment]
What's the reason for it being set to flutter.ndkVersion

**Comment by BenjaminHalko** [comment]
OK, so `flutter.ndkVersion` is an NDK version that is supposedly the "most compatible" with the flutter framework. I think currently it is `23.something`

**Comment by BenjaminHalko** [comment]
But you are fully allowed to change the NDK version if there is a version mismatch.

**Comment by oSumAtrIX** [comment]
Why would the intl dependency rely on an NDK not most compatible with Flutter. I think downgrading the library makes more sense

**Comment by BenjaminHalko** [comment]
Unfortunately, `flutter_localizations` depends on intl `0.19.0` and `flutter_localizations` is from the flutter SDK. That said other plugins that have been recently updated/added are `skeletons` and `file_picker`. So it might be one of those...

**Comment by BenjaminHalko** [comment]
It could have been a dependency of a package that got upgraded...

File: android/app/build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     compileSdk 34
-    ndkVersion flutter.ndkVersion
+    ndkVersion "26.1.10909125"
```
Comment (validcube) [inline android/app/build.gradle:27]: This should be safe to remove, see

https://github.com/flutter/flutter/issues/139427#issuecomment-1989024985

**Review by validcube — APPROVED** [review]
🥞 Approved!

**Pull request merged**